### PR TITLE
Improvement/JQA-10020/adding logs to error

### PR DESCRIPTION
**Pull request by vmrvictor**
1. added logs (parser.log)
2. added temp folder where the converter will store temporal files (step in the middle between original and final conversion to test cases) 
3. updated README with troubleshooting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,20 +3,22 @@
   "version": "1.0.0",
   "description": "Migration utility to convert test cases exported from Zephyr and XRay to a format supported by TM4J - Test Management for Jira",
   "scripts": {
-    "start": "node app/index.js"
+    "start": "node app/index.js",
+    "debug": "node --inspect app/index.js"
```
Comment (vmrvictor) [inline package.json:7]: added also debug mode

File: README.md [hunk 2]
```diff
@@ -71,4 +71,29 @@ If you don't want to map anything, please leave the fields empty or with default
...
+8) Import each converted file in Zephyr Scale using the default import option "Test Management for Jira".
+
+## Troubleshooting ##
+
+### Unexpected close tag ###
+Supposing that you have a file named myInput.xml on /input folder.
+During npm start you get the following error (or similar):
+```
+It was not possible to convert the content printed above
+look at the line specified by the error message:Unexpected close tag
```
Comment (juro994) [inline README.md:83]: This look at the line sentence shouldn't be part of the ``` rich text block, or should it?

Comment (vmrvictor) [reply README.md:83]: ah sorry this is part of the log message but in the console they will see without it

Comment (juro994) [reply README.md:83]: aah ok. Neverming. I see that you've added that log in the `app/testCaseXMLParser.js`. So ignore my previous comment :D

**Pull request merged**